### PR TITLE
For the button to add the activeOpacity attribute interface

### PR DESCRIPTION
**Pull request by moxiaohui**
For the button to add the activeOpacity attribute interface, let the button click effect is more flexible.
Example of the second button Add the attribute test.
Add a description of the property the Readme.md.

### CODE REVIEW

**Comment by alvaromb** [comment]
Thanks for the PR @moxiaohui!

Can you try to squash your commits?

**Comment by matteomunaretto** [comment]
Hi there, any news about that? Thanks.

File: Example/button/Example.js [hunk 1]
```diff
@@ -3,6 +3,12 @@ import Button from 'apsl-react-native-button'
 
 export default class Example extends React.Component {
   render () {
+    var onPressProps;
```
Comment (alvaromb) [inline Example/button/Example.js]: Better use `let` if possible.

**Comment by alvaromb** [comment]
Yes! sorry folks @moxiaohui @matteomunaretto 

I can merge this later today.

**Comment by alvaromb** [comment]
Merging.

**Pull request merged**